packages/webpack/src/config/base.js

```diff
@@ -10,14 +10,16 @@ import TerserWebpackPlugin from 'terser-webpack-plugin'
 import WebpackBar from 'webpackbar'
 import env from 'std-env'
 
-import { isUrl, urlJoin } from '@nuxt/utils'
+import { createTimeHash, isUrl, urlJoin } from '@nuxt/utils'
 
 import PerfLoader from '../utils/perf-loader'
 import StyleLoader from '../utils/style-loader'
 import WarningIgnorePlugin from '../plugins/warning-ignore'
 
 import { reservedVueTags } from '../utils/reserved-tags'
 
+const BUILD_HASH = createTimeHash()
```

Comment:
Move into env ?

Reply:
won't the hash be different for server and client then?

Reply:
Right, but we need a way to generate new hash for different builds

Reply:
Oh, so you would have different hashes for server and client *intentionally*?

Reply:
No, I mean your concern is right, we need to find some way else